setup.py

```diff
@@ -7,22 +7,26 @@
 
 import awesome
 
+HERE = os.path.abspath(os.path.dirname(__file__))
+
 # Package meta-data.
 NAME = 'awesome-finder'
 DESCRIPTION = 'A TUI based awesome curated list finder'
 URL = 'https://github.com/mingrammer/awesome-finder'
 EMAIL = 'k239507@gmail.com'
 
 # What packages are required for this module to be executed?
-REQUIRED = [
-    'requests'
-]
-
-here = os.path.abspath(os.path.dirname(__file__))
+REQUIRED = []
+with io.open(os.path.join(HERE, 'requirements.txt'), encoding='utf-8') as f:
+    for line in f:
```

Comment:
You should use `f.readlines()` for reading the contents line by line from the file descriptor

Reply:
Why would `readlines()` be preferable to this method? I find this more concise and memory efficient than loading the whole file at once (even if memory efficiency isn't a real deal here given the file size) , but if there's good reasons why you think `readlines()` would be better here, I'll be happy yo fix.

Reply:
@mattoufoutu Oh sorry for that. I thought that the `for line in f` could not work, but it was my misapprehension. It'd be better one. Thanks :)